.github/workflows/footprint-tracking.yml

```diff
@@ -22,7 +22,7 @@ concurrency:
 
 jobs:
   footprint-tracking:
-    runs-on: ubuntu-22.04
+    runs-on: zephyr-runner-linux-x64-4xlarge
```

Comment:
FWIW, there exists `zephyr-runner-linux-x64-xlarge`, which has 4 vCPUs instead of 16 -- we likely do not need 16 vCPUs for these workflows.

Reply:
last time I checked those were not working.

Reply:
this one, yes. But we also have a PR related action.

Reply:
True. And is it really useful, btw? If I'm not mistaken it's not setting the diff as a "build output" (hence making it cumbersome to quickly go and look for the information in the job), and, more importantly, it's not really failing (or commenting on the PR, or whatever else we deem suitable) even if it does detect an increase?

Reply:
@stephanosio  

> FWIW, there exists `zephyr-runner-linux-x64-xlarge`, which has 4 vCPUs instead of 16 -- we likely do not need 16 vCPUs for these workflows.

change to that runner now, and as suspected, it does not launch...

Reply:
> @stephanosio
> 
> > FWIW, there exists `zephyr-runner-linux-x64-xlarge`, which has 4 vCPUs instead of 16 -- we likely do not need 16 vCPUs for these workflows.
> 
> changed to that runner now, and as suspected, it does not launch...

Isn't this because the use of these runners is somewhat protected? You want to avoid that anybody can use them.
I typically see a 
if: github.repository_owner == 'zephyrproject-rtos'
before using them
Not sure how this is enforced though.